src/libraries/Microsoft.Diagnostics.Tracing.EventSource.Redist/src/Microsoft.Diagnostics.Tracing.EventSource.Redist.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks>net461-Windows_NT</TargetFrameworks>
     <Nullable>enable</Nullable>
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
Because we are adding the `[SupportedOSPlatform("windows")]` for all windows build I needed to add the attributes for projects not having it, please let me know if there is other way